openedx/core/djangoapps/credentials/management/commands/tests/test_notify_credentials.py

```diff
@@ -127,7 +127,6 @@ def test_hand_off(self, mock_grade_cert_change, mock_grade_interesting, mock_pro
         call_command(Command(), '--start-date', '2017-02-01')
         self.assertEqual(mock_grade_cert_change.call_count, 2)
         self.assertEqual(mock_grade_interesting.call_count, 2)
-        self.assertEqual(mock_program_changed.call_count, 2)
```

Comment:
Remove it's mocking call as well.

Reply:
I think i removed the wrong mocking call in earlier reviewed PR. Call to be removed was the first one. `self.assertEqual(mock_grade_cert_change.call_count, 2)`  and not `self.assertEqual(mock_program_changed.call_count, 2)`